### ISSUE TITLE
Fails to build for nfft 3.3.0

### ISSUE THREAD

**Issue by mikapfl** [ghisvail/pyNFFT#49]
Hi,

pynfft fails to build for the latest nfft (released less than a month ago) due to nfft3utils.h not being generated anymore.
Using nfft 3.2.4 it builds without problems.

Cheers,

Mika

**Comment by ghisvail** [comment]
Yes, there are multiple factors as to why pyNFFT will fail to build with 3.3 onwards.

One is the nfft3utils header, which is no longer shipped. That can be fixed easily.

Another is the move to multi-precision, so libnfft3 -> libnfft3_double.

I am intending to work on a fix for pyNFFT v2.0, but cannot commit to a precise ETA.

Thanks for reporting this @mikapfl 

Best regards,
Ghis

**Comment by cancan101** [comment]
Fixed here: https://github.com/ghisvail/pyNFFT/pull/58?
